### PR TITLE
Added new scripts for verification of GetURLS.

### PR DESCRIPTION
**Pull request by istoimenova**
The scripts are based on different urls in sdl_preloaded_pt.json regarding registered / non-registered applications / defaults.

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        if(is_correct[i] == false) then
+          self:FailTestCase("url: "..data.result.urls[i].url.." is not correct. Expected: "..endpoints[i].url)
+        end
+      end
+    end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_PTU_GetURLs_AppRegistered()
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua]: @istoimenova, Could you explain please what is the purpose of this test step?

File: test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      for i = 1, #is_correct do
+        if(is_correct[i] == false) then
+          self:FailTestCase("url: "..data.result.urls[i].url.." is not correct. Expected: "..endpoints[i].url)
+        end
+      end
+    end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua]: @istoimenova, Please remove 2nd "Test" group

File: test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua (hunk 1 above)
Comment (dboltovskyi) [inline test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua]: @istoimenova, Could you explain please what is the purpose of this test step?

Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/055_ATF_P_TC_HMI_sends_GetURLs_one_app_registered_urls_3default.lua]: @dboltovskyi, the step checks results in Local PT, as this is not in scope of BB testing step is removed from the scripts 055, 056, 057

**Pull request merged**